.ci/ignore_pip_conflicts.txt

```diff
@@ -0,0 +1,11 @@
+notebooks/109-performance-tricks/109-latency-tricks.ipynb  # ultralytics<=8.0.178
+notebooks/109-performance-tricks/109-throughput-tricks.ipynb  # ultralytics<=8.0.178
+notebooks/123-detectron2-to-openvino/123-detectron2-to-openvino.ipynb # detectron2@https://github.com/facebookresearch/detectron2
+notebooks/122-quantizing-model-with-accuracy-control/122-yolov8-quantization-with-accuracy-control.ipynb  # ultralytics==8.0.43
+notebooks/212-pyannote-speaker-diarization/212-pyannote-speaker-diarization.ipynb  # pyannote-audio@https://github.com/eaidova/pyannote-audio
+notebooks/230-yolov8-optimization/230-yolov8-instance-segmentation.ipynb  # ultralytics==8.0.43
+notebooks/230-yolov8-optimization/230-yolov8-keypoint-detection.ipynb  # ultralytics==8.0.159
+notebooks/230-yolov8-optimization/230-yolov8-object-detection.ipynb  # ultralytics==8.0.43
+notebooks/256-bark-text-to-audio/256-bark-text-to-audio.ipynb  # torch==1.13
+notebooks/267-distil-whisper-asr/267-distil-whisper-asr.ipynb  # optimum-intel@https://github.com/eaidova/optimum-intel
+notebooks/254-llm-chatbot/254-llm-chatbot.ipynb  # nncf@https://github.com/openvinotoolkit/nncf
```

Comment:
that already resolved should be, i return nncf 2.7 back couple of days ago, but it required to have openvino-nightly